lib/friendly_shipping/services/rl/serialize_create_bol_request.rb

```diff
@@ -66,17 +66,19 @@ def serialize_reference_numbers(reference_numbers)
             }.compact
           end
 
-          # @param [Range] pickup_time_window
+          # @param [FriendlyShipping::Services::RL::BOLOptions] options
           # @return [Hash, nil]
-          def serialize_pickup_request(pickup_time_window)
+          def serialize_pickup_request(options)
+            pickup_time_window = options.pickup_time_window
```

Comment:
Super minor nit: I think we can get away without assigning this variable if we return from this method